README.md

```diff
@@ -214,6 +215,10 @@ APIs
 - [FreeDB](http://www.freedb.org/) - Look up CD information using the freed database.
 - [QuantOneMusic](http://quantonemusic.com/) - Answers the need for digital music data.
 
+### Movies
+
+- [OMDB](https://www.omdbapi.com/) - The OMDb API is a free web service to obtain movie information. ![Open Source](https://raw.githubusercontent.com/Abhishaker17/Awesome-APIs/master/opensource.png?token=AH5Y2imVR5ukgG9533IEqXnP-RJWxVJQks5W4PuNwA%3D%3D "Open Source")
```

Comment:
This link won't work anymore when you delete your repository. Better [host it on GitHub](https://vinkla.com/2015/upload-images-github/) if you want it to always remain available.